news/_posts/translation/2020-09-09-do-you-speak-prestashop-august-2020.md

```diff
@@ -0,0 +1,134 @@
+ 
+#### Top 10 translators in August:
+ 
+| |Translator | Language | # Words
+|-|---------- | -------- | ----------------
+ 1. | PrestaServicePro | Azerbaijani | 4,945
+ 2. | M.Mirena (mmirena) | Albanian | 2,098
+ 3. | Girdman.com (anargaraezov)  | Azerbaijani | 1,961
+ 4. | mirmal  | Esperanto  | 1,336
+ 5. | Gabriel Tenita (ggedamed)  | Romanian | 1,051
+ 6. | Martin (lord_dark)  | Polish | 902
+ 7. | Milan Kolka (milankolka)  | Esperanto | 586
+ 8. | jbonello | Maltese | 553
+ 9. | Rivo Zängov (eraser)  | Estonian | 507
+10. | Enrique García (quiquegarcia)   | Spanish, Mexico | 283
+ 
+ 
+#### Top 6 proofreaders in August:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+ 1. | M.Mirena (mmirena)  | Albanian | 2,302
+ 2. | Girdman.com (anargaraezov)  | Azerbaijani | 1,179
+ 3. | Enrique García (quiquegarcia)  | Spanish, Mexico | 1,143
+ 4. | Gabriel Tenita (ggedamed) | Romanian | 1,138
+ 5. | Monika (monikaraciunaite)  | Lithuanian | 190
+ 6. | Stanislas Yordanov (StanProg)  | Bulgarian | 16
+ 
+Congrats, and welcome to our new top contributors! :clap:
+ 
+Remember, you can see who's been contributing to our translation project thanks to the [Translators page](http://translators.prestashop.com/).
+ 
+ 
+## Complete translations
+ 
+#### Fully translated languages
+ 
+At the end of August 2020, PrestaShop 1.7.7 was fully available (= 100% translated and validated) in **10 languages**:
+ 
+| Chinese Traditional | Danish | Dutch | French |
+| Italian  | Lithuanian | Portuguese, Brazil | Romanian |
+| Slovak | Spanish | Swedish |
+ 
+ 
+## Languages with the best evolution
+ 
+In comparison with July 2020, the following languages had the best progress thanks to the translation community:
+ 
+* **Esperanto** (+4% to reach 40% translated)
+* **Azerbaijani** (+4% to reach 17% translated)
+* **Albanian** (+4% to reach 39% translated)
+ 
+![Best translation progress for August 2020](/assets/images/2020/09/build-crowdin-progress-august20.png)
+
+A huge thank you to all the contributors!
+ 
+Of course, this is highlighting the languages that made some progress with new translations; but it doesn't mean that the languages that aren't mentioned here aren't active. Indeed, some editing and rewriting could be going on, but the percentage of translation wouldn't be modified (since it's working on strings that are already translated). So let's not forget about the work of other proofreaders! Thanks to you too!
+ 
+ 
+## Languages that need (more) proofreaders
+ 
+A translated string will not be available in PrestaShop as long as it is not validated. For this reason, it’s important to keep a high level of validated strings vs. translated strings, to make sure everyone benefits from the latest translations! :fire:
+ 
+At the end of August 2020, some languages would still benefit from some proofreading:
+ 
+* **Spanish, Argentina** (100% translated vs 13% validated).
```

Comment:
it means this is time to find new proofreaders for this language. aslo, it might be worth it to contact the current proofreader and ask if they are still active or if they want to stop.